test/integration/routes_test.rb

```diff
@@ -5,7 +5,12 @@
 class RoutesTest < ActionDispatch::IntegrationTest
   fixtures :all
 
-  test "test spectrums choose route" do
+   "test spectrums choose route" do
```

Comment:
Hey @PaulinaGadea looks like you have accidentally removed the test word here which is why the tests are failing can you add it back? Rest all looks great :100:  Thanks :v:

Reply:
Hi @Tlazypanda, thank you so much! I hope it's okay now